### PR TITLE
PG-1013 Updated install steps for pg_tde

### PR DESCRIPTION
**Pull request by nastena1606**
PG-1013

### Description
The extension is now part of the server package, no need to install additional ones. Repo is no longer experimental

modified:   documentation/docs/apt.md
	modified:   documentation/docs/yum.md

### CODE REVIEW

**Review by shahidullah79 — CHANGES_REQUESTED** [review]
For pg_tde beta user need to enable 'testing' repo instead of experimental and release repos.  
 $ sudo percona-release enable ppg-{{pgversion17}} experimental
  $ sudo percona-release enable ppg-{{pgversion17}} testing

File: documentation/docs/apt.md [hunk 1]
```diff
@@ -37,10 +40,10 @@ You need the `percona-release` repository management tool that enables the desir
 
 4. Enable the Percona Distribution for PostgreSQL repository
 
-    Percona provides [two repositories](repo-overview.md) for Percona Distribution for PostgreSQL. We recommend enabling the Major release repository to timely receive the latest updates. Since the `tde_heap` access method is still in the experimental stage, the `pg_tde` package is currently available from the experimental repository.
+    Percona provides [two repositories](repo-overview.md) for Percona Distribution for PostgreSQL. We recommend enabling the Major release repository to timely receive the latest updates. Since the extension is in the {{release}} stage, enable the testing repository.
 
     ```{.bash data-prompt="$"}
-    $ sudo percona-release enable ppg-{{pgversion17}} experimental
+    $ sudo percona-release enable ppg-{{pgversion17}} testing
     ```
```
Comment (ImTheKai) [inline documentation/docs/apt.md:47]: Won't this be "main" after the release?

Comment (shahidullah79) [reply documentation/docs/apt.md:47]: Currently for Beta we are using this. IMO after GA it will be move to main repo.

Comment (dutow) [reply documentation/docs/apt.md:47]: Won't we release the 17.2 package into the main repo, and just say that pg_tde in it is beta?

Comment (shahidullah79) [reply documentation/docs/apt.md:47]: Yes it is part of server package then we need main/release repo to be configured.

Comment (nastena1606) [reply documentation/docs/apt.md:47]: So what repo is the right one for 17.2 and the Beta?

Comment (maqeel75) [reply documentation/docs/apt.md:47]: 17.2

Comment (nastena1606) [reply documentation/docs/apt.md:47]: Updated

**Review by maqeel75 — APPROVED** [review]

**Review by shahidullah79 — APPROVED** [review]

**Review by maqeel75 — APPROVED** [review]

**Review by dutow — APPROVED** [review]

**Pull request merged**